src/components/Seperator/Seperator.jsx

```diff
@@ -2,21 +2,23 @@ import React from 'react';
 import styled from '@emotion/styled';
 import PropTypes from 'prop-types';
 
-const Seperator = ({ width, height, type }) => {
-  return <Wrapper type={type} width={width} height={height} />;
+const Seperator = ({ width, height, type, margin }) => {
+  return <Wrapper type={type} width={width} height={height} margin={margin} />;
 };
 
 const Wrapper = styled.hr`
-  width: ${({ type }) => type === 'vertical' && '0.1rem'};
-  height: ${({ type, height }) => (type === 'horizon' && '0.1rem') || height};
-  color: ${({ theme }) => theme.colors.lighterGray};
-  border-bottom: 0;
+  width: ${({ type }) => type === 'vertical' && '0.05rem'};
```

Comment:
제가 width를 빼먹었나봐요, 에드워드 미안하지만 수정 부탁드려도 될까요?
```suggestion
  width: ${({ type, width }) => type === 'vertical' && '0.05rem' || width};
```

Reply:
아래와 같이 코드 작성했습니다.

```suggestion
  width: ${({ type, width }) => type === 'vertical' && (width || '0.05rem'});
```